client/tsconfig.json

```diff
@@ -17,5 +17,6 @@
 		"jsx": "react-jsx",
 		"types": ["node", "jest"]
 	},
-	"include": ["src"]
+	"include": ["src"],
+	"exclude": ["node_modules", "dist", "**.test.ts", "**.test.tsx"]
```

Comment:
Should we exclude tests for typescript?

Reply:
I dont know how to works tests on client side manage by react, i just find it in a web article.